x/ibc-transfer/client/cli/tx.go

```diff
@@ -40,24 +37,34 @@ func GetTransferTxCmd(cdc *codec.Codec) *cobra.Command {
 			sender := cliCtx.GetFromAddress()
 			srcPort := args[0]
 			srcChannel := args[1]
-			destHeight, err := strconv.Atoi(args[2])
+			receiver := args[2]
+
+			coins, err := sdk.ParseCoins(args[3])
 			if err != nil {
 				return err
 			}
 
-			// parse coin trying to be sent
-			coins, err := sdk.ParseCoins(args[4])
+			timeoutHeight, err := strconv.ParseUint(args[4], 10, 64)
 			if err != nil {
-				return err
+				return fmt.Errorf("invalid timeout height: %w", err)
+			}
+
+			timeoutTimestamp, err := strconv.ParseUint(args[5], 10, 64)
+			if err != nil {
+				return fmt.Errorf("invalid timeout timestamp: %w", err)
 			}
 
-			msg := types.NewMsgTransfer(srcPort, srcChannel, uint64(destHeight), coins, sender, args[3])
+			msg := types.NewMsgTransfer(
+				srcPort, srcChannel, coins, sender, receiver, timeoutHeight, timeoutTimestamp,
+			)
 			if err := msg.ValidateBasic(); err != nil {
 				return err
 			}
 
 			return authclient.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
 		},
 	}
+	cmd.Flags().Uint64(flagTimeoutHeight, types.DefaultPacketTimeoutHeight, "timeout height for fungible token transfer packet")
```

Comment:
This still has to be relative (we have to query the height of the dest chain's client)

Reply:
hm, you can't do that for default values. I'll try a workaround

Reply:
hmm, I'd need to (under the hood):

1. Query the channel from the source port and channel ids
2. Query the connection with the channel fields from 1.
3. Query the client-id and retrieve the latest height in order to add it to the default timeout height
 
It sounds like a bit overhead for me tbh. Up to you. cc: @AdityaSripal @colin-axner

Reply:
Maybe a default timeout timestamp is easier? If we just use a default height of `1000`, though, that won't work (in most cases the packet will time-out immediately), so if we don't query better to require a height.

Reply:
> Maybe a default timeout timestamp is easier? 

I don't understand. There is a default timestamp already `0` (i.e disabled)?

> If we just use a default height of 1000, though, that won't work (in most cases the packet will time-out immediately), so if we don't query better to require a height.

The user will have to query the client themself and then add the `dest height + desired timeout height` for it to work properly. Maybe setting a default timeout height of `0` (disabled) is better in this case. As long as it's documented, it should be ok.

cc: @jackzampolin

Reply:
What this tells me is that there should be a querier for finding what `client-id` supports a given `channel-id`/`port-id`